packages/planetmagfields.yml

```diff
@@ -0,0 +1,4 @@
+repo: AnkitBarik/planetMagFields
+section: domain specific plotting library
```

Comment:
```suggestion
section: domain specific libraries
```

Sorry, we need to improve the docs on exactly what these are supposed to be (partly they are still in flux...)

Reply:
Yes, I understand. I wasn't completely sure how to classify it, but this sounds good. Thanks! :)